docs/development-process.md

```diff
@@ -0,0 +1,48 @@
+## Firefly Development process
+
+- _Note on examples_
+    - This example shows the `dev` branch, if you are working on a rc branch always use the rc branch instead of `dev`
+    - This example development branch will be `firefly-123456-table-sort`
+- Branch naming scheme 
+    - Branch Scheme: `Ticket Name` - `word or two description`
+    - Such As: Ticket fix as a table feature might be: `Firefly-123456-table-sort`
+
+### Process
+1. Create a branch using the naming scheme above from the `dev` branch
+    - Create the branch from the base branch
+        - goto base branch: `git checkout dev`
```

Comment:
I normally also do a `git pull` here to ensure I start with the latest as well.

Reply:
yes, I should add that.